libs/template/src/lib/let/let.directive.ts

```diff
@@ -215,7 +234,7 @@ export class LetDirective<U> implements OnInit, OnDestroy {
   }
 
   ngOnDestroy() {
-    this.subscription.unsubscribe();
+    this.subscription?.unsubscribe();
```

Comment:
We could also initialize it `  protected subscription: Unsubscribable = new Subscription();`

Reply:
Done.